### PR TITLE
Update 1st UX (FE & BE)

### LINKED ISSUE #69: Update store for new data

### PR DESCRIPTION
**Pull request by brabbuss**
### Description
- *BACKEND* I modified `mutators` and `resolvers` to take new fields when creating a new user
- add new user profile fields on signup
- did not include description in 1st ux but added placeholder text to profile page

### Closes issue(s)
#70 

### Changes include
- [ ] Bugfix (non-breaking change that solves an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

### CODE REVIEW

File: code/web/src/modules/user/Profile.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ import { Helmet } from "react-helmet";
 import { Link } from "react-router-dom";
 import { ProfileModal } from "../../ui/modal/index";
 import EditProfileForm from "../../ui/form/EditProfileForm";
+// impor {placeholder} from './profile-placeholder.svg'
```
Comment (mdflynn) [inline code/web/src/modules/user/Profile.js:9]: Clean this up

File: code/web/src/modules/user/api/actions.js [hunk 2]
```diff
@@ -84,7 +84,8 @@ export function register(userDetails) {
     return axios.post(routeApi, mutation({
       operation: 'userSignup',
       variables: userDetails,
-      fields: ['id', 'name', 'email']
+      fields: ['id', 'name', 'email', "streetAddress", "city", "state", "zip", "country", "image", "description"]
+      // fields: ['id', 'name', 'email']
```
Comment (mdflynn) [inline code/web/src/modules/user/api/actions.js:88]: Let's clean this up as well

**Review by mdflynn — APPROVED** [review]
Much needed functionality to the sign up

**Review by araspaas — COMMENTED** [review]
lookin good.

**Pull request merged**